### ISSUE TITLE
Selected scopes are not included in token network-request for `client_credentials` authorization flow

### ISSUE THREAD

**Issue by JakobTeske** [mrin9/RapiDoc#299]
Foremost, much respect for building rapidoc 👍 

I have an API, which uses `client_credentials` as authentication flow. Example extract from openapispec yaml:

```
securitySchemes:
  exampleOuath2Flow:
        type: oauth2
        description: >-
          You authorize requests, by providing client credentials.
        flows:
          clientCredentials:
            tokenUrl: 'http://localhost:8080/token'
            scopes:
              testscope: "always required"
              user: "optional"

```
This is correctly displayed in the authentication section of Rapidoc, but if I select for example the `user` scope. The scope is not added to the scopes parameter in the network request:

<img width="1422" alt="Bildschirmfoto 2020-09-11 um 17 14 26" src="https://user-images.githubusercontent.com/12516608/92942931-678cd580-f452-11ea-8634-2d2843346fb2.png">

As my Token Gateway is expecting this scope, I do not get a valid token back.

I had a look at the code in `security-scheme-template.js` and saw that scopes are correctly added for `authorizationCode` and `implicit` authentication flows, but not for `client_credentials`.

To support scopes for `client_credentials` flows aswell, I would like to create a PR. Ist that ok?

**Comment by mrin9** [comment]
sure, if its missing, the PR will be a huge help

**Comment by JakobTeske** [comment]
Hey I opened a PR, which adds the desired functionality: #301. Would be great, if you could take a look :)

**Comment by mrin9** [comment]
thanks for the PR
closed by bed98f8ecca5d26f55997be11761fea4f6cff630
